text/0000-general-fund.md

```diff
@@ -0,0 +1,101 @@
+
+# Community-level explanation
+[community-level-explanation]: #community-level-explanation
+
+## Appropriate and inappropriate uses of the Grin Development Fund
+
+Although spending decisions will always be made on a case-by-case basis, it is understood that the Grin Development Fund is to be used mainly to finance the development needs of the Grin project and the advancement of Grin/Mimblewimble as a whole. 
+
+There is scope for interpretation as to what this actually means, but the following presents (non-exhaustive) guidelines as to the types of activities that would be considered appropriate to finance from the fund.
+
+* Activities related to the continuing development needs of the Grin code-base and related projects under the GitHub [mimblewimble](github.com/mimblewimble) organization. This can include:
+   * Fixed-term development or project-management contracts
+   * Infrastructure and other incidental costs
+   * Security audit costs
+   * Costs incurred for any legal matters that may arise for either the project or individual contributors 
+arising directly from their work in good faith on the Grin project
+* Running budgets for Grin teams (with funds spent at their own discretion)
+* Technological research in areas that may be relevant to Grin in the future
+* Cryptographic research related to Grin or Mimblewimble in general
+* Supporting non-profit, community-driven and inclusive promotional efforts for the Grin Project (e.g. Grincon) 
+* Charitable donations toward important causes relevant to the Grin or cryptocurrency communities as a whole
+
+While the guidelines above give plenty of flexibility in how funds can be employed toward the advancement of the Grin project, certain types of funding requests will not be considered:
+
+* Marketing or promotional 'offers' where a party is taking a profit or gains some financial benefit to promote Grin (regardless of how the 'opportunity' is presented)
+* Cryptocurrency speculation or any kind of investment
+
+## Request for Funding Process
+
+All requests for funding should be made in the [Governance Section](https://forum.grin.mw/c/governance) of the Grin forum. Topics should be clearly titled "Request for Funding: " followed by a short description of the request.
+
+There is no set template for what the contents of the funding request should include, however a better reception can be attained by including:
+
+* The exact amount requested, what the funding covers and for what time period (if applicable)
+* A clear description of what will be provided to the Grin project in return, including:
+    * Details of any work or research that will be performed during the period
+    * The expected deliverable, if applicable (for a research project, for instance)
+* Background information on the person making the request
+
+Amounts can be stated in local currency equivalents, if desired, with the exact Cryptocurrency amount determined by the rate at the time of transfer.
```

Comment:
To avoid a repeat of the recent forum fiasco, I propose narrowing this to requests being made in USD or BTC only.

Reply:
I don't think it was the change of exchange rate that led to the 'forum fiasco' as you call it, it was more a failure from my side to clearly outline in my original request that the change would result in a net raise in USD terms.

I don't mind enforcing USD / BTC (and / GRIN) only in requests, but it's not clear to me what problem it solves: In my particular example, if that would be the case, I would still state my preferred GBP amount on the request, convert it to USD according to the x-rate of that day, and then have the equivalent of that paid out in BTC amount once the request is approved. And I reckon I would then still need to field questions about "why last quarter you asked for $x and now you ask for $y", to which the answer then would be: x-rate fluctuations.

But perhaps this makes things a bit easier for the project, as it outsources fiat currency calculations to the requestor and makes it easier for people to review the amounts? I don't know if this is a big issue. If somebody raises a request in some obscure currency without some expression of what this equates to in a currency that people can relate to, it's not helping their chances getting support for their request, so there's no incentive to do so.

Perhaps there could be some sub-set of international currencies in that case (USD, GBP, EUR, JPY, CNY perhaps?), to make it clear that we're welcoming people from all parts of the world to ask in whatever (major) currency they prefer. But then it might become a question of "Hey why can't I ask in Swiss francs, or Canadian dollars, or Russian Roubles"? And yeah, why can't they? They will still only get paid in the currency of what's available in the fund, currently BTC or GRIN.

Reply:
> it's not clear to me what problem it solves
It feels more transparent. Most people in the crypto industry are already used to dealing in terms of USD and BTC (and yes, we should accept requests in GRIN). 

> I reckon I would then still need to field questions
Probably, yes. But at least then the questions would be about why you're asking for a salary increase, and not why you're trying to *hide* the salary increase. That's much better than the conversation beginning with the fundee being accused of wrongdoing. It's merely a change in salary at that point, which can easily be justified by "cost of living increase/decrease".

Reply:
Maybe just something like I am requesting ______ (currency of choice) per month, which at the time of writing is equivalent to _____ BTC & $_____ USD per month.  Just mandate a reference to BTC & USD at the time of writing, which might be different when paid out.  I am not sure if it would help or hurt, but you could also consider paying out the entire approved sum at one time rather than monthly, which would reduce accounting headaches in a volatile market.